daml-script/test/daml/ScriptTest.daml

```diff
@@ -171,6 +172,15 @@ partyIdHintTest = do
   dan <- allocatePartyWithHint "dan" (PartyIdHint "dan")
   pure (carol, dan)
 
+listKnownPartiesTest : Script ([PartyDetails], Party)
+listKnownPartiesTest = do
+  before <- listKnownParties
+  p <- allocateParty "myparty"
```

Comment:
Maybe a more specific name like `ListKnownPartiesTestParty` is less prone to accidental collisions with future tests?

Reply:
This is the display name not the party id. Collisions shouldn’t matter here.

Reply:
Oh, right. Sleepy me... Pardon!